neo-cli/CLI/MainService.Logger.cs

```diff
@@ -71,43 +93,78 @@ private void OnLog(string source, LogLevel level, object message)
             lock (syncRoot)
             {
                 DateTime now = DateTime.Now;
-                var log = $"[{now.TimeOfDay:hh\\:mm\\:ss\\.fff}] {message}";
-
-                if (Settings.Default.Logger.ConsoleOutput)
+                var log = $"[{now.TimeOfDay:hh\\:mm\\:ss\\.fff}]";
+                if (_showLog)
                 {
                     var currentColor = new ConsoleColorSet();
-
+                    var messages = message is string msg ? Parse(msg) : new[] { message.ToString() };
+                    ConsoleColorSet logColor;
+                    string logLevel;
                     switch (level)
                     {
-                        case LogLevel.Debug: DebugColor.Apply(); break;
-                        case LogLevel.Error: ErrorColor.Apply(); break;
-                        case LogLevel.Fatal: FatalColor.Apply(); break;
-                        case LogLevel.Info: InfoColor.Apply(); break;
-                        case LogLevel.Warning: WarningColor.Apply(); break;
+                        case LogLevel.Debug: logColor = DebugColor; logLevel = "DEBUG"; break;
+                        case LogLevel.Error: logColor = ErrorColor; logLevel = "ERROR"; break;
+                        case LogLevel.Fatal: logColor = FatalColor; logLevel = "FATAL"; break;
+                        case LogLevel.Info: logColor = KeyColor; logLevel = "INFO"; break;
```

Comment:
```suggestion
                        case LogLevel.Info: logColor = InfoColor; logLevel = "INFO"; break;
```
Ok for me, except the green all over the window. Could you set the info log color as white?
![1655971816730](https://user-images.githubusercontent.com/43694095/175250963-898cdfb3-d68f-4b3f-845d-0fe9da34a6e3.png)